persons.json

```diff
@@ -0,0 +1 @@
+["{\"json_class\":\"OpenStruct\",\"t\":{\"name\":\"Pierre Jardim\",\"id\":901,\"className\":\"Student\",\"parent_permission\":{\"parent_permission\":false},\"age\":19}}", "{\"json_class\":\"OpenStruct\",\"t\":{\"name\":\"Teacher X\",\"id\":556,\"className\":\"Teacher\",\"specialization\":\"Astrophysics\",\"age\":0}}", "{\"json_class\":\"OpenStruct\",\"t\":{\"name\":\"Khan\",\"id\":463,\"className\":\"Teacher\",\"specialization\":\"Doctor\",\"age\":34}}", "{\"json_class\":\"OpenStruct\",\"t\":{\"name\":\"Naveed\",\"id\":93,\"className\":\"Student\",\"parent_permission\":{\"parent_permission\":true},\"age\":45}}", "{\"json_class\":\"OpenStruct\",\"t\":{\"name\":\"Hameed\",\"id\":46,\"className\":\"Teacher\",\"specialization\":\"coder\",\"age\":29}}"]
```

Comment:
- [ ] If the persons.json file is empty the app crashes, kindly update it with something similar to 
```ruby
    return [] unless File.exists?("people.json")
```

![image](https://user-images.githubusercontent.com/62705947/195442510-429b0449-4c88-4bc9-a3d9-1ee60fa28d20.png)